packages/apps/job-launcher/server/src/common/constants/index.ts

```diff
@@ -47,4 +47,7 @@ export const HCAPTCHA_MIN_SHAPES_PER_IMAGE = 1;
 export const HCAPTCHA_MAX_SHAPES_PER_IMAGE = 1;
 export const HCAPTCHA_MINIMUM_SELECTION_AREA_PER_SHAPE = 5;
 
-export const HCAPTCHA_NOT_PRESENTED_LABEL = 'Not presented';
\ No newline at end of file
+export const HCAPTCHA_NOT_PRESENTED_LABEL = 'Not presented';
+
+export const HCAPTCHA_REPO_URI = "http://reporacle.yellow.svc.cluster.local:5000";
+export const HCAPTCHA_RO_URI = "http://recoracle:5000/rocheck";
```

Comment:
this is not a valid url, it might be valid in local environment using docker, set those variables using env variables or kvstore

Reply:
Let use .env variables for now as for rest services (CVAT, Fortune).